packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -112,7 +105,7 @@ export class QuoteRequestor {
         const _opts: RfqtRequestOpts = { ...constants.DEFAULT_RFQT_REQUEST_OPTS, ...options };
         assertTakerAddressOrThrow(_opts.takerAddress);
 
-        const ordersWithStringInts = await this._getQuotesAsync<SignedOrder>( // not yet BigNumber
+        const firmQuotes = await this._getQuotesAsync<FirmQuote>( // not yet BigNumber
```

Comment:
`RFQTFirmQuote`?

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2582/commits/ad868af96e13f3efa68d5836d898c53a6a5f8252